tests/test_BaseISCSI.py

```diff
@@ -85,6 +86,7 @@ def test_attach_tgt_present_path_found(self, mock_init_paths):
         self.subject.attach(self.sr_uuid)
 
     @mock.patch('BaseISCSI.BaseISCSISR._initPaths', autospec=True)
+    @mock.patch('sm.core.xs_errors.XML_DEFS', 'libs/sm/core/XE_SR_ERRORCODES.xml')
```

Comment:
This shouldn't be required if it's done at the class level?

Reply:
Ah missed some removals of an interim attempt.